extensions/plugins/system/redcore/redcore.php

```diff
@@ -316,6 +316,18 @@ public function onBeforeCompileHead()
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.js']);
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.min.js']);
 			}
+
+			// Remove permission.js. Specially for Joomla 3.6
+			if ($isAdmin && version_compare(JVERSION, '3.6.0', '>='))
```

Comment:
Is that correct? But when I try to check on my localhost. It's just return true, when I'm in redCORE view or redCORE's base extensions view (redFORM, redITEM,...) not on other Joomla Extension. (com_installer, com_content,...)